pyflakes/checker.py

```diff
@@ -768,6 +768,13 @@ def handleNodeLoad(self, node):
                     # iteration
                     continue
 
+            if (name == 'print' and
+                    isinstance(scope.get(name, None), Builtin)):
+                parent = self.getParent(node)
+                if (isinstance(parent, ast.BinOp) and
+                        isinstance(parent.op, ast.RShift)):
+                    self.report(messages.InvalidPrintSyntax, node)
```

Comment:
The reported error node should be the `>>`, not the `print`, as it is the `>>` which is incompatible, and under Python 3 there is no ability to change to using the print operator so that the `>>` can become valid.

Reply:
Turns out this was impossible to do as the >> node position is not available in the ast.